integration_tests/tests/integrity/integrity_campaign_country_report.sql

```diff
@@ -0,0 +1,67 @@
+{{ config(
+        tag="fivetran_validations",
```

Comment:
```suggestion
        tags="fivetran_validations",
```
The integrity tests aren't actually running because they're looking for `tags`